demo/api.md

```diff
@@ -132,18 +132,43 @@ Sets a persistent error state (e.g. an error state returned from the server). Th
 
 To give a higher limit you can bind a date to the `maxDate` property.
```

Comment:
Let's preserve a very basic non-functional example like we had before. While the new example is great - it complicates understanding of the basic use of the maxDate attribute.

```
<auro-datepicker maxDate="06/16/1980">
  <span slot="label">Choose a date</span>
</auro-datepicker>
```